app/services/metrics.py

```diff
@@ -19,30 +17,35 @@ def get_areas_by_defined_area(area_type, area_id):
         # TODO: Implement service
         return ""
 
-    def get_areas_by_polygon(polygon):
-        # TODO: read raster from STAC
-        raster_cloud_path = "https://staccatalog.blob.core.windows.net/cog-test/Colombia_pp-2015_12_31-pp_2011_2015.tif"
+    def get_areas_by_polygon(
+        polygon: PolygonGeometry, metric_id: str
+    ) -> list[dict[str, float]]:
+        collection_url = find_collection_url(settings.stac_url, metric_id)
+
+        items_url = get_collection_items_url(collection_url)
```

Comment:
I still don't see why it's necessary to have the collection url first and then the items, why don't just get the items directly?